post_office/__init__.py

```diff
@@ -1,4 +1,7 @@
-VERSION = (3, 3, 1)
+from os.path import dirname, join
+
+with open(join(dirname(__file__), 'version.txt'), 'r') as fh:
+    VERSION = eval(fh.read())
```

Comment:
Just a minor nitpick, can we just read the file and treat it as string and split the version codes by `,`? I try to avoid using eval whenever I can.

Reply:
I fully agree that using `eval` is an absolute no-go for untrusted sources. But here we are in full control of the file `version.txt`, so it's a "dangerous"  as putting any code into `__init__.py`, `version.py` or anywhere else.

Parsing the file, means to sort out all kinds of nasty problems, such as interpreting numbers correctly and stripping whitespace. And I'm not sure if that step is optimized properly, so it could be that it will be performed each time one loads any submodule from `post_office`.

Other proposal would be to use [ast.literal_eval](https://docs.python.org/3/library/ast.html#ast.literal_eval):

> Safely evaluate an expression node or a string containing a Python literal or container display. The string or node provided may only consist of the following Python literal structures: strings, bytes, numbers, tuples, lists, dicts, sets, booleans, and None.
This can be used for safely evaluating strings containing Python values from untrusted sources without the need to parse the values oneself. It is not capable of evaluating arbitrarily complex expressions, for example involving operators or indexing.

Reply:
Ok, let's use `ast.literal_eval` then.